### ISSUE TITLE
Markdownlint failing due to end of life Node.js version

### ISSUE THREAD

**Issue by chris-cheshire** [nf-core/tools#1423]
### Description of the bug

Markdownlint is failing with the following internal error

```
Run markdownlint .
/opt/hostedtoolcache/node/10.2[4](https://github.com/nf-core/cutandrun/runs/5358906901?check_suite_focus=true#step:5:4).1/x64/lib/node_modules/markdownlint-cli/markdownlint.js:107
  return files.flat().map(file => ({
               ^

TypeError: files.flat is not a function
    at prepareFileList (/opt/hostedtoolcache/node/10.24.1/x64/lib/node_modules/markdownlint-cli/markdownlint.js:107:16)
    at Object.<anonymous> (/opt/hostedtoolcache/node/10.24.1/x64/lib/node_modules/markdownlint-cli/markdownlint.js:2[5](https://github.com/nf-core/cutandrun/runs/5358906901?check_suite_focus=true#step:5:5)1:15)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:[6](https://github.com/nf-core/cutandrun/runs/5358906901?check_suite_focus=true#step:5:6)53:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:5[8](https://github.com/nf-core/cutandrun/runs/5358906901?check_suite_focus=true#step:5:8)5:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:1[9](https://github.com/nf-core/cutandrun/runs/5358906901?check_suite_focus=true#step:5:9))
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Error: Process completed with exit code 1.
```

After some research, I have found it is because the newer version `0.31.0` of markdownlint is not compatible with the EOL node.js version 10.

Upgrading node to 14 in `linting.yml` fixes the issue.

**Comment by chris-cheshire** [comment]
I see this may already be fixed in `dev` actually...

**Comment by mashehu** [comment]
This is fixed in 2.23dev https://github.com/nf-core/tools/pull/1398
